src/devices/wholeBodyDynamics/app/wholebodydynamics-icub-external-six-fts-sim.xml

```diff
@@ -1,75 +1,73 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <!DOCTYPE devices PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
 
-<devices robot="@WBD_YARP_ROBOT_NAME@" build="1">
-    <device name="wholebodydynamics" type="wholebodydynamics">
-        <param name="axesNames">(torso_pitch,torso_roll,torso_yaw,neck_pitch, neck_roll,neck_yaw,l_shoulder_pitch,l_shoulder_roll,l_shoulder_yaw,l_elbow,l_wrist_prosup,l_wrist_pitch,l_wrist_yaw,r_shoulder_pitch,r_shoulder_roll,r_shoulder_yaw,r_elbow,r_wrist_prosup,r_wrist_pitch,r_wrist_yaw,l_hip_pitch,l_hip_roll,l_hip_yaw,l_knee,l_ankle_pitch,l_ankle_roll,r_hip_pitch,r_hip_roll,r_hip_yaw,r_knee,r_ankle_pitch,r_ankle_roll)</param>
-        <param name="modelFile">model.urdf</param>
-        <param name="fixedFrameGravity">(0,0,-9.81)</param>
-        <param name="defaultContactFrames">(l_hand,r_hand,root_link,l_sole,r_sole,l_upper_leg,r_upper_leg,l_elbow_1,r_elbow_1)</param>
-        <param name="imuFrameName">imu_frame</param>
-        <param name="useJointVelocity">true</param>
-        <param name="useJointAcceleration">true</param>
+<device xmlns:xi="http://www.w3.org/2001/XInclude" name="wholebodydynamics" type="wholebodydynamics">
```

Comment:
We don't need to define `xmlns:xi="http://www.w3.org/2001/XInclude"` here since we don't use `xi:include`.